build/bootstrap/bootstrap-ubuntu-20.04.sh

```diff
@@ -0,0 +1,64 @@
+#! /bin/bash
+
+# bootstrap scripts will exit immediately if a command exits with a non-zero status
+set -e
+
+echo "Setting up development environment for do-client"
+
+# Various development machine tools
+apt-get update
+apt-get install -y build-essential g++ gdb gdbserver git wget
+apt-get install -y python3 cmake ninja-build rpm
+
+# Open-source library dependencies
+apt-get install -y libboost-all-dev libgtest-dev libproxy-dev libmsgsl-dev libssl-dev uuid-dev
+
+# Install cpprest dependencies
+# libssl-dev also required but installed above because plugin uses libssl-dev directly
+apt-get install -y zlib1g-dev
+
+# Cpprestsdk 2.10.2 is the latest publicly available version on Ubuntu 18.04
```

Comment:
This comment references 18.04. Intentional?

Reply:
Thanks for the catch, modifying comment